### PR TITLE
Removed common trace/debug functionality

### PR DESCRIPTION
**Pull request by matthewloring**
Functionality moved into https://www.npmjs.com/package/@google/cloud-diagnostics-common

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,7 +17,8 @@
 'use strict';
 
 var OpaqueSpan = require('./lib/opaque-span.js');
-var Logger = require('./lib/logger.js');
+var CloudCommon = require('@google/cloud-diagnostics-common');
```
Comment (ofrobots) [inline index.js]: Capitalization here incorrectly suggests that this is a constructor / class name. Use `cloudCommon`, or just `common` instead.

File: index.js [hunk 2]
```diff
@@ -17,7 +17,8 @@
...
+var Logger = CloudCommon.logger;
```
Comment (ofrobots) [inline index.js]: ditto. This should be `var Logger = common.Logger;` as the latter is actually a class, so it should exported from that module as such.

Comment (matthewloring) [inline index.js]: Tracking logger capitalization in common here: https://github.com/matthewloring/cloud-diagnostics-common-nodejs/issues/1

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**